.github/workflows/test-package-build.yml

```diff
@@ -51,8 +51,10 @@ jobs:
 
           # Set step outputs
           wheel_path=$(find dist -type f -name "*.whl")
+          wheel_name=$(basename $wheel_path)
           wheel_size=$(stat -c %s $wheel_path)
           echo "::set-output name=wheel-path::${wheel_path}"
+          echo "::set-output name=wheel-name::${wheel_path}"
```

Comment:
`wheel_name` is consumed here:

https://github.com/mlflow/mlflow/pull/4067/files#diff-02817d741ec741866d08356819c7ed9eb21aa4dbafa62f75164ec6bf160df4a2R70-R75

Reply:
Should this line be:
```
echo "::set-output name=wheel-name::${wheel_name}"
```
?

Reply:
Thanks for the catch! Fixed it.